src/ensembl/src/content/app/custom-download/components/paste-or-upload/PasteOrUpload.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import PasteOrUpload from './PasteOrUpload';
+import faker from 'faker';
+import Input from 'src/shared/input/Input';
+import ImageButton from 'src/shared/image-button/ImageButton';
+
+const onChange = jest.fn();
+const onRemove = jest.fn();
+
+describe('<PasteOrUpload/>', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  let wrapper: any;
+  const defaultProps = {
+    value: null,
+    onChange,
+    onRemove
+  };
+
+  it('renders without error', () => {
+    wrapper = () => {
+      mount(<PasteOrUpload {...defaultProps} />);
+    };
+    expect(wrapper).not.toThrow();
+  });
+
+  it('displays the text "Paste data or Upload file"', () => {
+    wrapper = mount(<PasteOrUpload {...defaultProps} />);
+
+    expect(wrapper.text()).toBe('Paste data or Upload file');
+  });
+
+  it('does not display the text "Paste data or Upload file" when a value is passed', () => {
+    wrapper = mount(<PasteOrUpload {...defaultProps} value={'foo'} />);
+
+    expect(wrapper.text()).not.toBe('Paste data or Upload file');
```

Comment:
do we expect any `wrapper.text()` at all?

Reply:
Updated the test to check if the text is ''.